Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Remove /usr from the initscript's PATH
 All utilities reside on /sbin. Furthermore, the declaration of
 /usr/sbin makes lintian complain about a missing dependency on $remote_fs,
 which is strictly not necessary.

Last-Update: 2014-06-13
Forwarded: no
--- a/scripts/drbd
+++ b/scripts/drbd
@@ -31,7 +31,7 @@
 UDEV_TIMEOUT=10
 ADD_MOD_PARAM=""
 
-PATH=/usr/sbin:/sbin:$PATH
+PATH=/sbin:/bin
 
 if [ -f $DEFAULTFILE ]; then
   . $DEFAULTFILE
